New View Claims Page

sloan-seamus opened the pull request:
Revamped the View Claims page.
Claims are now being sorted by their status.

Work in progress. Unsure of how to sort upon load up.

Commit:
Revamped the View Claims page.
Claims are now being sorted by their status.

Work in progress. Unsure of how to sort upon load up.

dsifford commented on frontend/src/app/view-claims/view-claims.component.ts:
```diff
@@ -27,6 +25,10 @@ export class ViewClaimsComponent implements OnInit {
 
   private claims: Protos.Claim[] = [];
+  private claims0: Protos.Claim[] = [];
+  private claims1: Protos.Claim[] = [];
```
![no](https://i.giphy.com/media/POql6zsXZbmcE/giphy.webp)

sloan-seamus replied:
Agreed...

dsifford commented on frontend/src/app/view-claims/view-claims.component.ts:
```diff
@@ -56,8 +54,22 @@ export class ViewClaimsComponent implements OnInit {
-  openSnackBar() {
-
-
+  sortClaims(num): void{
```
Delete this entire method and instead just filter by status on click.

So, if they clicked status 2....

```ts
this.visibileClaims = this.claims.filter(claim => claim.status === 2)
```

sloan-seamus replied:
I left the method there, but instead it returns a sorted array of the claims. The for loop in the .html goes through the sorted list.

Commit:
Finished the claims sorting on View Claims.
Fixed an issue with footer not sticking to bottom when more than 6 claims

Added routerLink to Ford Logo in right corner.
(Disclaimer: There's a blue rectangle around the logo after clicking it. Unsure of what that is)

sloan-seamus committed:
Merge branch 'master' into new-view-claims

sloan-seamus commented on frontend/src/app/view-claims/view-claims.component.html:
```diff
@@ -1,15 +1,68 @@
+</ly-tabs>
+
+<div>
   <ng-template #noClaims>
```
Possible change here would be to say "No claims are currently awaiting settlement options" when there are claims, but no claims in that status.

dsifford committed:
Merge branch 'master' into new-view-claims

dsifford committed:
Merge branch 'master' into new-view-claims

dsifford committed:
Merge branch 'master' into new-view-claims
